pyiron/base/generic/hdfio.py

```diff
@@ -1231,19 +1232,18 @@ def create_working_directory(self):
         if not os.path.isdir(self.working_directory):
             os.makedirs(self.working_directory)
 
-    def create_object(self, class_name, **qwargs):
+    def import_class(self, class_name):
         """
-        Internal function to create a pyiron object
+        Import given class from fully qualified name and return class object.
 
         Args:
-            class_name (str): name of a pyiron class
-            **qwargs: object parameters
+            class_name (str): fully qualified name of a pyiron class
 
         Returns:
-            pyiron object: defined by the pyiron class in class_name with the input from **qwargs
+            class object: class object of the given name
         """
         internal_class_name = class_name.split(".")[-1][:-2]
-        if internal_class_name in self._project.job_type.job_class_dict.keys():
+        if internal_class_name in self._project.job_type.job_class_dict:
```

Comment:
I do not think this can work, with a dictionary

Reply:
Why not? Iterating over a dict iterates over the keys by default.

Reply:
I just did not know about this one.